internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -2576,6 +2632,7 @@ func expandKubernetesClusterNetworkProfile(input []interface{}) (*managedcluster
 		NetworkPlugin:   utils.ToPtr(managedclusters.NetworkPlugin(networkPlugin)),
 		NetworkMode:     utils.ToPtr(managedclusters.NetworkMode(networkMode)),
 		NetworkPolicy:   utils.ToPtr(managedclusters.NetworkPolicy(networkPolicy)),
+		KubeProxyConfig: expandKubernetesClusterKubeProxyConfig(config["kube_proxy"].([]interface{})),
```

Comment:
We should only set this in the payload in the create if it's been explicitly specified in the config.